pkgs/development/tools/parsing/tree-sitter/default.nix

```diff
@@ -49,6 +51,8 @@ in rustPlatform.buildRustPackage {
   pname = "tree-sitter";
   inherit src version cargoSha256;
 
+  buildInputs = lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
```

Comment:
Not sure if this should be `darwin.apple_sdk.frameworks.Security` or `darwin.Security`; I have no way to test.

Reply:
What do other packages do?

Reply:
Or maybe: what prompted you to add it if you don’t have a MacOS machine?

Reply:
I can testify that the build process indeed fails without it on MacOS. This dependency seem to be typical for Rust packages, like `bat` or `git-and-tools.delta`.

Reply:
@NickHu sorry I didn't clarify earlier, this is how I've normally seen this accomplished:
* add a parameter called `Security` at the top of `tree-sitter/default.nix`
* set `buildInputs = lib.optionals stdenv.isDarwin [ Security ];` in `tree-sitter/default.nix`
* update `top-level/all-packages.nix` with:

```
tree-sitter = callPackage ../development/tools/parsing/tree-sitter {
   inherit (darwin.apple_sdk.frameworks) Security;
};
```

You can ping me when the change is made and I can test that it builds on macOS :)

Reply:
@ipetkov ping

Reply:
@NickHu 4d8adccab694baf4d413e8feef308c9b1da2d507 builds on macOS 👍